tests/test_api/test_auth.py

```diff
@@ -152,6 +172,31 @@ def test_get_user_deleted(client):
     assert rv.status_code == 404
 
 
+def test_get_user_non_existent(client):
+    user_id = 1
+
```

Comment:
Can't add `model.User.query.get.return_value = None` as it affects [`test_create_cluster()`](https://github.com/resource-hub-dev/rhub-api/blob/338a0ae4818620c9b2816e7a26473dd6aa1867c1/tests/test_api/test_lab/test_cluster.py#L420)